app/views/idv/verify_info/show.html.erb

```diff
@@ -102,6 +102,16 @@ locals:
             toggle_label: t('forms.ssn.show'),
           ) %>
     </div>
+<% if IdentityConfig.store.doc_auth_ssn_controller_enabled %>
```

Comment:
I would indent this whole block and line it up with the `</div>` above? If the worry was about diff noise, we can always suggest reviews hide whitespace
```suggestion
    </div>
    <% if IdentityConfig.store.doc_auth_ssn_controller_enabled %>
```